projects/klippa/ngx-enhancy-forms/src/lib/elements/radio/radio.component.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../../style/themes/default/variables';
+@import '../../../style/mixins';
```

Comment:
@wouter-willems we need to add the style sources in angular.json so you can just `@import "variables"` and `@import "mixins"`.

Not this ticket though.